graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,35 @@
+// import { Context } from '../../@types/helpers'
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+// import { isAdminOrThrow } from '../../helpers/isAdmin'
+
+export const modules = async () => {
+  return prisma.module.findMany({
+    include: {
+      author: true
+    }
+  })
+}
+
+export const addModule = async (_parent: void, args: MutationAddModuleArgs) => {
+  //isAdminOrThrow(req) will add after, easier for testing
```

Comment:
I am not understanding this. Why are we not using `isAdminOrThrow`, and instead have it commented out?

Reply:
@anthonykhoa is right, dont put testing code into production!